properties/utils.py

```diff
@@ -43,6 +43,12 @@ def func_wrapper(self):
 
     return func_wrapper
 
+def filter_dict(cls, fields):
```

Comment:
I co-opted this function into the 0.2.3 release: https://github.com/3ptscience/properties/blob/master/properties/utils.py#L54